ipn/ipnlocal/local.go

```diff
@@ -304,6 +308,14 @@ func NewLocalBackend(logf logger.Logf, logid string, store ipn.StateStore, diale
 		loginFlags:     loginFlags,
 	}
 
+	// for now, only log sockstats on unstable builds
+	if version.IsUnstableBuild() {
```

Comment:
given how much we've trimmed down the log messages, I'm now actually feeling a little more comfortable going ahead and turning this on for all devices.  What do you think @mihaip ?

Reply:
hahaha, wow... you went aggressive with single character names:
```
{"t":1678419626985,"d":99,"i":"en0","s":{"0":{"tx":55,"rx":710}}}
{"t":1678419627185,"d":100,"i":"en0","s":{"1":{"tx":332}}}
{"t":1678419627285,"d":99,"i":"en0","s":{"1":{"tx":1170,"rx":364}}}
{"t":1678419627484,"d":100,"i":"en0","s":{"1":{"tx":876,"rx":721}}}
{"t":1678419627685,"d":100,"i":"en0","s":{"1":{"tx":45,"rx":12453}}}
{"t":1678419627885,"d":99,"i":"en0","s":{"1":{"tx":1032,"rx":1018},"6":{"tx":113},"7":{"tx":365},"8":{"tx":960,"rx":352}}}
{"t":1678419627984,"d":99,"i":"en0","s":{"1":{"rx":58},"8":{"tx":600,"rx":528}}}
{"t":1678419628084,"d":99,"i":"en0","s":{"1":{"tx":1670},"2":{"tx":268},"6":{"tx":73},"7":{"tx":351},"8":{"tx":320,"rx":704}}}
```

What's the compression ratio of this now? This is now dominated by double-quotes, the repeated prefix for a timestamp, and the interface name, all of which are highly compressible.

Reply:
It's even more compact than that now, since @mihaip suggested I remove the "tx" and "rx".  It now looks like:

```json
{"t":1678487681796,"d":98,"s":{"1":[1064,934],"6":[113,0],"7":[365,0],"8":[960,220]}}
{"t":1678487681895,"d":100,"s":{"8":[600,440]}}
{"t":1678487681995,"d":99,"s":{"1":[1746,35],"6":[73,0],"7":[351,0],"8":[320,528]}}
{"t":1678487682095,"d":100,"s":{"2":[629,4688],"8":[0,572]}}
{"t":1678487682196,"d":100,"s":{"1":[0,88],"8":[0,220]}}
{"t":1678487682296,"d":99,"s":{"3":[3295,0],"8":[0,88]}}
```

I haven't actually tried compressing it yet, but yeah it would probably be pretty good.

Reply:
I suppose if you really want to squeeze bytes, you could turn the pairs into triples, and move the key inside, e.g.,

```
{"1":[1064,934],"6":[113,0],"7":[365,0],"8":[960,220]}
[[1,1064,934],[6,113,0],[7,365,0],[8,960,220]]
```

but that may be too obfuscated to be worthwhile.

Reply:
closing the loop on the original comment here... discussed in team meeting this morning, and we'll keep it only for unstable builds right now.